client/app/lib/onboarding.coffee

```diff
@@ -0,0 +1,111 @@
+# Local class Step
+class Step
+    # Retrieves properties from config Step plain object
+    constructor: (step) ->
+        if step
```

Comment:
If we could keep this as generic as possible and with the minimum dependancies, I think it's better for the reusability. I don't see a big interest to use underscore here.